### PR TITLE
fix a resizing issue that's been annoying me for a while

### PR DESCRIPTION
**Pull request by Lyxal**

### CODE REVIEW

**Comment by cgccuser** [comment]
Not working for me, the code box is always about 2 lines tall, and once I have enough letters to reach the second line, there's a scroll bar, but the box doesn't resize.

**Review by cgccuser — APPROVED** [review]
Feels weird to have this heuristic-y thing, but it does work better than what we have currently so 👍.

**Review by PyGamer0 — APPROVED** [review]
idk js but looks fine

**Review by razetime — APPROVED** [review]

File: static/main.js [hunk 1]
```diff
@@ -262,8 +262,9 @@ function initCodeMirror() {
         dummy.value = elem.doc.getValue()
         elem.setSize(
             null,
-            Math.max(dummy.scrollHeight - 5, elem.getTextArea().dataset.baseHeight || 27)
```
Comment (pxeger) [inline static/main.js:265]: does this mean the `dummy` element is no longer needed at all?

**Review by pxeger — CHANGES_REQUESTED** [review]

File: static/main.js (hunk 1 above)
Comment (Lyxal) [reply static/main.js:265]: No. It's still needed to set the other styles of the codemirror boxes.

**Pull request merged**